### ISSUE TITLE
Choppers RTB but don't land

### ISSUE THREAD

**Issue by NZF-JDWang** [SceptreOfficial/Simplex-Support-Services#35]
Seen this a few times now, when you tell the transport choppers to RTB sometimes they will fly back to base and just hover there until you tell them RTB once more, then they will land.

**Comment by SceptreOfficial** [comment]
I've seen this happen and I don't know why it's broken only for RTB. I'll try to figure it out.

**Comment by EpsilonCommunity** [comment]
I've noticed this happening too

**Comment by SceptreOfficial** [comment]
The issue is that every other request that interrupts an active request gets immediately cancelled. This only happens on a dedicated server. And for the life of me, I cannot figure out why this is happening.

I found a workaround that seems to have solved the problem (executing the cancellation code on the next frame).

If this problem persists after the next update, let me know.

**Comment by Kiskadee-dev** [comment]
This is still an issue, multiple requests sometimes will mess with the last one and will require that we repeat the call.

For example, requesting a move and later changing it to land will make it move to the last order but won't land until we repeat the call.

**Comment by Vertexbingo** [comment]
Im playing using mods, the biggest ones being ALiVE and LAMBS fsm. If I disable LAMBS fsm in-game using Zeus the RTB function works perfectly. Its strange, idk if you guys are using AI mods but its almost like if its the AI's pathfinding.

**Comment by SceptreOfficial** [comment]
da41e7c8267984452dd918a0d421e72641a29008

**Comment by SceptreOfficial** [comment]
Test with the new version and create a new issue if it still occurs.
